Resources/Private/Fusion/Controller/Backend/ExportDefinition/Index.fusion

```diff
@@ -9,11 +9,15 @@ prototype(PunktDe.Form.Persistence:Backend.ExportDefinition.Index) < prototype(N
     exportDefinitionEditorApp = Neos.Fusion:ResourceUri {
         path = 'resource://PunktDe.Form.Persistence/Public/ExportDefinitionEditorApp/build/static/js/main.js'
     }
+    config = ${Configuration.setting('PunktDe.Form.Persistence.exporter')}
```

Comment:
Alle drei Zeilen können in eine gefasst werden. Falls in mehrere Zeilen, bitte der Variable einen spezifischeren Namen geben. Beispiel: exporterConfig

Reply:
unbenannt in exporterTypes und den code eingekürzt.